scripts/ansible/embericelm.yml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: validators-embreicelm 
+  gather_facts: False
+  pre_tasks:
+    - raw: test -e /usr/bin/python || (apt -y update && apt install -y python-minimal)
+    - setup: # aka gather_facts
+  become: yes
+  become_user: root
+  vars_files:
+    - group_vars/ee-refresh-env.yml
+  roles:
+    - logstash
```

Comment:
I think it's a good idea to have it on those systems, but the ci is about the software and this is more like infrastructure. Not sure if it's beneficial here.

Reply:
Split it out into a generic ansible role?

Reply:
IMO a general role for setting up validator nodes could include this as this is not part of the software deployment and only has to be run once - or changed by devops. What is the intention of putting it here?

Reply:
Logstash has been moved to the devops repo